config/locales/fr.yml

```
@@ -848,6 +848,10 @@ fr:
       default: "%d %B %Y %R"
       long: "%d %B %y %H:%M"
       human: "%A %-d %B à %Hh%M"
+      long_date: "%A %d %B %Y"
+      long_datetime: "%A %d %B %Y à %H:%M"
```

pas de leading 0 pour les n° de jour en format texte donc :  
```suggestion
      long_date: "%A %-d %B %Y"
      long_datetime: "%A %-d %B %Y à %H:%M"
```